roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -15,7 +15,7 @@ class LoggingCheck(OpenShiftCheck):
     logging_namespace = "logging"
 
     def is_active(self):
-        logging_deployed = self.get_var("openshift_hosted_logging_deploy", type=bool, default=False)
+        logging_deployed = self.get_var("openshift_hosted_logging_deploy", convert=bool, default=False)
```

Comment:
`convert=bool` reads well, though `convert=str` is a bit strange since we perhaps expect the input to be a string already?! I liked `type`, though it is the name of a builtin function and may make pylint complain ;-)